api/src/app.ts

```diff
@@ -146,6 +147,7 @@ export const build = async (
     secret: SESSION_SECRET,
     rolling: false,
     saveUninitialized: false,
+    // cookieName: 'jwt_access_token', NOTE: Confirm if we want to use different cookie name for session token
```

Comment:
If we're going with `sessionId` then the mobile app needs to be updated to use the new cookie name. There are two ways of doing that:
1. We update the code to check and extract both the old and new name cookies and set them both to same value when sending requests. The advantage with this approach is the app update can be released earlier than the deployment of new API.
2. Or we can use Firebase Remote Config to and keep the new API stuff behind a flag which gets enabled right when the new API gets deployed. Now I'm not sure how long it will take for the change to go out in this case.

Reply:
I prefer the first approach. Thanks.